python/pyspark/sql/functions.py

```diff
@@ -4450,7 +4573,7 @@ def _unresolved_named_lambda_variable(*name_parts):
     )
 
 
-def _get_lambda_parameters(f):
+def _get_lambda_parameters(f: Callable) -> Any:
```

Comment:
Can the return type be more specific instead of `Any`?

Reply:
Updated to `ValuesView`